app/api/orders.py

```diff
@@ -127,9 +127,9 @@ def before_create_object(self, data, view_kwargs):
                 free_ticket_quantity += 1
 
         if not current_user.is_verified and free_ticket_quantity == len(data['ticket_holders']):
-            raise UnprocessableEntity(
-                {'pointer': '/data/relationships/order'},
-                "Unverified user cannot place free orders"
+            raise ForbiddenException(
+                {'pointer': '/data/relationships/user'},
+                "Unauthorized Access"
```

Comment:
Revert this. However, there should be another field 'code' which will be a machine readable error code like `unverified-user` so that clients can take appropriate action